package.json

```diff
@@ -16,25 +16,33 @@
     "lib/"
   ],
   "scripts": {
-    "build": "browserify -d -r ./index.js:hydrolysis -o hydrolysis.js",
-    "build:watch": "watch 'npm run build' ./lib",
-    "release": "browserify -r ./index.js:hydrolysis -o hydrolysis.js",
+    "init": "typings install",
+    "clean": "rm -rf node_modules/ typings/ lib/ && find src | grep \\.js$ | xargs rm",
```

Comment:
Not at all thrilled unless it also got us faster builds (which admittedly should be possible, save probably for the final browserify step).